src/test/resources/county-court-judgment/by-instalments.json

```diff
@@ -0,0 +1,29 @@
+{
+  "defendant": {
+    "type": "individual",
+    "title": "Dr.",
+    "name": "John Smith",
+    "address": {
+      "line1": "52",
+      "line2": "Down Street",
+      "city": "Manchester",
+      "postcode": "DF1 3LJ"
+    },
+    "serviceAddress": {
+      "line1": "52",
+      "line2": "Down Street",
+      "city": "Manchester",
+      "postcode": "DF1 3LJ"
+    },
+    "email": "j.smith@example.com"
+  },
+  "paymentOption": "INSTALMENTS",
+  "paidAmount": 0,
+  "repaymentPlan": {
+    "remainingAmount": 1000,
+    "firstPayment": 100,
+    "instalmentAmount": 100,
+    "firstPaymentDate": "2100-10-10",
```

Comment:
Lol

Reply:
Valid date, I didn't want to have a date that may be in past before the project is finished ;)